### ISSUE TITLE
Checkboxes with the same name are set all with first value

### ISSUE THREAD

**Issue by crabnky** [FThompson/FormPersistence.js#12]
Hi.
I have few checkboxes with the same name, but different values:
```
<input type="checkbox" name="test[]" value="1"/>
<input type="checkbox" name="test[]" value="2"/>
```
It seems like all of them gets status of the first one - all are checked if first one is checked, no one is checked when first one is not checked.
When I check the values stored I see:
```
test[]: Array
0: false
1: true
```
so it is saved properly (first one uchecked, second one checked) - but when form is loaded both are unchecked.

**Comment by FThompson** [comment]
Hey thanks for letting me know about this bug! I found the issue and I'll push and publish a fix shortly.

**Comment by FThompson** [comment]
Published v2.0.5 with a bug fix for checkbox array inputs. Let me know if you continue to have issues. Thanks!

**Comment by crabnky** [comment]
Thanks!
